javascript-create-module/templates/module/docker-compose.yml

```
@@ -21,10 +22,15 @@ services:
       test: ["CMD", "curl", "http://localhost:8080/start"]
       interval: 5s
       timeout: 1s
-      retries: 24
+      retries: 30
 
   postgres:
     image: postgres:16
     environment:
       POSTGRES_USER: jahia
       POSTGRES_PASSWORD: dbpassword
+    healthcheck:
+      test: ["CMD", "pg_isready"]
```

```suggestion
      test: ["CMD", "pg_isready", "-U", "jahia"]
```
I tested it, the -U option seems to be necessary, otherwise I have this in the logs:
```
postgres-1  | 2025-09-16 09:52:09.475 UTC [369] FATAL:  role "root" does not exist
postgres-1  | 2025-09-16 09:52:14.579 UTC [376] FATAL:  role "root" does not exist
postgres-1  | 2025-09-16 09:52:19.731 UTC [384] FATAL:  role "root" does not exist
postgres-1  | 2025-09-16 09:52:24.832 UTC [391] FATAL:  role "root" does not exist
postgres-1  | 2025-09-16 09:52:29.993 UTC [398] FATAL:  role "root" does not exist
```